### ISSUE TITLE
Terra-Menu Buttons are Inaccessible

### ISSUE THREAD

**Issue by emilyrohrbough** [cerner/terra-core#1585]
# Issue Description
Terra-menu's bounded menu's close/back buttons are not accessible. This will require internationalization to resolve this issue. The [translations from terra-action-header](https://github.com/cerner/terra-core/tree/master/packages/terra-action-header/translations) can be reused for this. 

## Issue Type

- [ ] New Feature
- [x] Enhancement
- [x] Bug
- [ ] Other

## Current Behavior
![screen shot 2018-06-11 at 10 04 17 am](https://user-images.githubusercontent.com/14099737/41239907-da56a82a-6d5e-11e8-9a37-0e1920877c4b.png)
